### PR TITLE
chore(happo): mirror puppeteer enabled and disabled tests

### PR DESCRIPTION
**Pull request by ozgurkececioglu**
[FX-2821]

### Description

Enabled all stories by default the same as `puppeteer` and disable the disabled ones as well

### How to test

- we should see new stories on happo report

### Development checks

- Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
-  Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
-  Annotate all `props` in component with documentation
-  Create `examples` for component
-  Ensure that deployed demo has expected results and good examples
-  Ensure that tests pass by running `yarn test`
-  Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
-  Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- Covered with tests

**Breaking change**

- codemod is created and showcased in the changeset
-  test alpha package of Picasso in StaffPortal

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2821]: https://toptal-core.atlassian.net/browse/FX-2821?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/picasso/src/Timeline/story/index.jsx [hunk 1]
```diff
@@ -18,8 +18,14 @@ page
...
-  .addExample('Timeline/story/Dates.example.tsx', 'Dates') // picasso-skip-visuals
+  .addExample('Timeline/story/Default.example.tsx', {
+    title: 'Default',
+    takeScreenshot: false
+  }) // picasso-skip-visuals
+  .addExample('Timeline/story/Dates.example.tsx', {
+    title: 'Dates',
+    takeScreenshot: false
+  }) // picasso-skip-visuals
   .addExample('Timeline/story/TrimLastConnector.example.tsx', {
```
Comment (separatio) [inline packages/picasso/src/Timeline/story/index.jsx:29]: This was also disabled. You missed it.

Comment (separatio) [reply packages/picasso/src/Timeline/story/index.jsx:29]: Fixed.

**Review by separatio — APPROVED** [review]
Manually inspected all changes. Looks good. :+1:

**Comment by denieler** [comment]
we have screenshots added for Props as well, but they are empty - https://happo.io/a/675/p/1189/compare/76ef440b815e98a8aaaa48ea668bc94c9c190279/5b96a354f0f346d3ff8f473c8caf4f165713d752?t=added#Accordion-Props

I think we need either remove them from Happo, either fill up with prop documentation 😃

**Review by denieler — APPROVED** [review]
All the rest looks 🎸 🚀 🎉

**Review by TomasSlama — APPROVED** [review]

**Pull request merged**